bonfire/processor.py

```diff
@@ -30,23 +30,82 @@ def _process_template(*args, **kwargs):
     return processed_template
 
 
-def _remove_resource_config(items):
-    for i in items:
-        if i["kind"] != "ClowdApp":
+def _is_trusted_config(value, regex, component_params, path):
+    """
+    Check for presence of a trusted param being used for the value.
+
+    A trusted parameter is defined as:
+    1. matching the expected regex pattern
+    2. the parameter value is set in the component's deploy config
+
+    Returns True if we determine this is a trusted value
+    """
+    in_params = False
+    match = re.match(regex, value)
+    if match and match.groups()[0] in component_params:
+        in_params = True
+
+    log.debug(
+        "value '%s', regex r'%s', matches=%s, in params=%s", value, regex, bool(match), in_params
+    )
+
+    if match and in_params:
```

Comment:
Could you just do 

```
return match and in_params
```

I'm thinking that would solve the problem of no `False` return in this method and would remove the conditional.